src/crewai/utilities/events/listeners/tracing/interfaces.py

```diff
@@ -0,0 +1,33 @@
+import json
+from datetime import datetime
+
+from crewai.cli.plus_api import PlusAPI
+from crewai.cli.authentication.token import get_auth_token
+from pydantic import BaseModel
+from .trace_batch_manager import TraceBatch
+from logging import getLogger
+
+logger = getLogger(__name__)
+
+
+class TraceSender(BaseModel):
+    """Trace sender for sending trace batches to the backend"""
+
+    def send_batch(self, batch: TraceBatch) -> bool:
+        """Print trace batch to console"""
+        try:
+            payload = batch.to_dict()
+
+            def datetime_handler(obj):
+                if isinstance(obj, datetime):
+                    return obj.isoformat() + "Z" if obj.tzinfo else obj.isoformat()
```

Comment:
Appending "Z" manually can be misleading unless you’re sure the datetime is in UTC.. I'm worried whether current approach is safer